src/domains/ecommerce-app/api/ecommerce-io/v1/post_transactions.xml.tpl

```diff
@@ -97,7 +97,7 @@
             <set-body>@{
               JObject requestBody = context.Request.Body.As<JObject>(preserveContent: true);
               requestBody["orderId"] = "ORDER_ID"; //To be removed since it is mandatory for transaction request body, but it should not be
-              requestBody["email"] = (String)context.Variables["email"];
+              requestBody["emailToken"] = (String)context.Variables["email"];
```

Comment:
Is it ok to update v1 too?

Reply:
We can leave it as it is given that it won't be used, what do you think?